cmake/common/check_configuration.cmake

```diff
@@ -16,7 +16,8 @@ macro(check_stdcxx)
     # Check C++11
     include(CheckCXXCompilerFlag)
     if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_COMPILER_IS_CLANG OR
-        CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+        CMAKE_CXX_COMPILER_ID MATCHES "Clang"OR
```

Comment:
```suggestion
        CMAKE_CXX_COMPILER_ID MATCHES "Clang" OR
```

Reply:
@MiguelCompany why is the build failing?

Reply:
it seems when I commit the changes you mentioned directly from github I was unable to sign them off. Had to make the changes locally and then recommit.